packages/caip/src/adapters/coingecko/utils.test.ts

```diff
@@ -69,6 +70,13 @@ const makePolygonMockCoingeckoResponse = () => ({
   platforms: {},
 })
 
+const makeGnosisMockCoingeckoResponse = () => ({
```

Comment:
could add fox asset on gnosis (polygon was forgotten as well) if we wanted, not super important, just noticed

Reply:
Will add as a follow-up for Polygon and Gnosis